src/navigation/navigation-events.ts

```diff
@@ -39,13 +47,14 @@ export function navigateToUrl(obj) {
     );
   }
 
-  const current = parseUri(window.location.href);
-  const destination = parseUri(url);
+  const current = new URL(window.location.href);
+  const destination = new URL(url, window.location.href);
```

Comment:
IE10 and/or 11 didn't support `new URL()` to correctly parse all strings. Since single-spa@7 doesn't support IE, I've removed the `parseUri` function in favor of `new URL`.

I am not totally sure that the behavior is the same for the two implementations, but I think it should be. The main edge cases to consider are relative urls - I believe using `window.location.href` as the base url will be equivalent to creating an anchor element with a relative url for the href